### PR TITLE
BZ#2021461: MetalLB Operator install errors

### PR DESCRIPTION
**Pull request by mikemckiernan**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=2021461

----

The updates are in [Installing from the OperatorHub using the CLI](https://deploy-preview-38864--osdocs.netlify.app/openshift-enterprise/latest/networking/metallb/metallb-operator-install#nw-metallb-installing-operator-cli_metallb-operator-install).

### CODE REVIEW

File: modules/nw-metallb-operator-initial-config.adoc [hunk 1]
```diff
@@ -18,7 +22,7 @@ After you install the Operator, you need to configure a single instance of a Met
 [source,terminal]
 ----
 $ cat << EOF | oc apply -f -
```
Comment (asood-rh) [inline modules/nw-metallb-operator-initial-config.adoc:24]: @mikemckiernan Most of it looks good. I just have a thought of not including prompt sign '$' where ever there is cat << EOF command to allows user to use the clipboard functionality to copy and paste the YAML.

Comment (mikemckiernan) [reply modules/nw-metallb-operator-initial-config.adoc:24]: Thanks @asood-rh, including the prompt is the Red Hat docs convention, but you've uncovered some kind of browser-specific bug that causes the prompt to get included in the paste.  I'll investigate that with my team.

**Review by kalexand-rh — COMMENTED** [review]
LGTM

**Pull request merged**